### PR TITLE
 Moving from database to firestore

### PR DESCRIPTION
**Pull request by devnook**

### CODE REVIEW

File: tools/import_products.js [hunk 1]
```diff
@@ -53,37 +53,56 @@ const parseProductLine = (record) => {
...
+  const query = collection.orderBy('__name__').limit(batchSize);
+  return new Promise((resolve, reject) => {
+    deleteQueryBatch(db, query, batchSize, resolve, reject);
+  });
+};
 
-const processRecord = (record) => (uploadProduct(parseProductLine(record)));
+const deleteQueryBatch = (db, query, batchSize, resolve, reject) => {
+  query.get().then((snapshot) => {
+    if (snapshot.size == 0) {
```
Comment (samdutton) [inline tools/import_products.js]: Nit: `===`

File: src/server/routes/category.js [hunk 2]
```diff
@@ -22,15 +22,18 @@ import categories from '../../data/categories';
 
 const category = (req, res, next) => {
   const categoryId = req.params.id;
-  fbAdmin.database().ref('/products')
-    .orderByChild('category')
-    .equalTo(categoryId)
-    .once('value')
+  fbAdmin.firestore().collection('products').where('category', '==', categoryId).get()
```
Comment (samdutton) [inline src/server/routes/category.js]: Should this be `'==='`?

**Comment by samdutton** [comment]
LGTM apart from minor nits.

File: src/server/routes/category.js (hunk 2 above)
Comment (devnook) [reply src/server/routes/category.js]: I don't think so, as it's a Firestore operator and not js:
"The operation string (e.g "<", "<=", "==", ">", ">=")
https://firebase.google.com/docs/reference/js/firebase.firestore.Query#where

**Pull request merged**